velox/connectors/Connector.h

```diff
@@ -113,6 +113,10 @@ class ConnectorInsertTableHandle : public ISerializable {
     return false;
   }
 
+  virtual std::string toString() const {
+    VELOX_NYI();
```

Comment:
Probably should be empty string or pure virtual

Reply:
Changed to use pure virtual